pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -715,6 +716,20 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get", "put", "update", "delete").URLs(templateapi.ServiceBrokerRoot + "/*").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-view",
+				Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-view": "true", "rbac.authorization.k8s.io/aggregate-to-admin": "true"}},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule(read...).Groups(metrics.GroupName).Resources("pods").RuleOrDie(),
```

Comment:
`metrics.GroupName` needs to be stored at the top group list like everything else

Reply:
Please provide me with a one sentence description of this resource so I can understand if it makes sense to aggregate into view.

Reply:
pods.metrics.k8s.io allows you to view CPU and memory usage for your pods.